src/course-outline/data/slice.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+const slice = createSlice({
+  name: 'courseOutline',
+  initialState: {
+    reindexLink: '',
+    lmsLink: '',
+    docsLinks: {
+      learnMoreOutlineUrl: '',
+      learnMoreGradingUrl: '',
+      learnMoreVisibilityUrl: '',
+    },
+  },
+  reducers: {
+    fetchOutlineIndexSuccess: (state, { payload }) => {
```

Comment:
Do we need a reducer for the download status and download process?

Reply:
Done